run_a11y.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+ENV=${1:-local}
+BROWSER=${2:-chrome}
```

Comment:
can we make this the same as the normal/new `run_tests.sh` scripts?

Reply:
Done :)